free-1.md

```diff
@@ -1,51 +1,73 @@
 them?_
 
+- Specifically, easily accessing average results and expected values that requires limited statistical knowledge (such as jargon). Currently most values when game designing can be calculated directed (especially simple ones) but more complex results (and expected value) is less obvious and usually just guessed. Ideally with this you could load up a python doc, or use a exterior DSL to quickly throw together expect value and results cleanly.  
+
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+- Unlike other tools, it should be a DSL specifically to utilize If -else terminology and useful ness. If I roll a 1, and a series of actions occur, then I want this dsl to account for that. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+Im a big Game Designer (ttrpg). It would help me do the things i want to do specifically. Honestly the more interesting question is if it would help anyone else. 
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Game Design result analysis 
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+- I think it could be an external thing, but even just a python plugin would be usable. Something quick and easy. Honestly anything that could quickly export visuals, so probably exterior is ideal 
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+- The program runs, calculates all the possibilities (running through each) then shows the expected value based on whats reported. Peferably in nice visuals 
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+- It should be easy to get expected values of dice rolls, and add contingencies for certian results. 
+- It should be difficult to edit the generated visuals, but it is possible just not the main use of the software 
+- I think keeping it hyper specific is better, so anything outside of math, loops and statistics is a no no 
+
 ### Related work
 
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+- I mean any statistics DSL is in this domain (python can do it generally) (R and mathematic etc is also definitely possible) And many sites have dice roll projections, but I think the goal is to give a middle ground between complete open statisitcal analysis and quick but powerful expirimentation. 
```

Comment:
I think this is a great idea for a DSL. The scope is pretty small, but there are a lot directions you could go with it. I was thinking about how this could potentially be used for cards as well. Another important thing to consider is how you are going to generate randomness. I don't know much about it, but I believe that there are many advanced methods of generating randomness besides random seeds.